incubator/jaeger/templates/_helpers.tpl

```diff
@@ -19,6 +31,7 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 {{- end -}}
 
+{{/*
```

Comment:
This comment was missing in the previous PR that got merged. I noticed it while I was resolving conflicts.